### DISCUSSION TITLE
ClientResponseError 521

### DISCUSSION THREAD

**Discussion by rutikthakre** [pocketbase/pocketbase#3092] [Q&A]
`{
  "url": "http://ec2-13-200-79-156.ap-south-1.compute.amazonaws.com:5000/api/collections/users/auth-with-password",
  "status": 521,
  "response": {},
  "isAbort": false,
  "originalError": {
    "url": "http://ec2-13-200-79-156.ap-south-1.compute.amazonaws.com:5000/api/collections/users/auth-with-password",
    "status": 521,
    "data": {}
  },
  "name": "ClientResponseError 521"
}`
i am using qwik js as front end and vercel for deployment and pocketbase is deployed on aws ec2 
when i am trying to log in with email and passowrd and my log in function is vercel edge function i am getting this error

**Comment by ganigeorgiev** [comment]
How are you connecting to your PocketBase server? Are you using HTTP or HTTPS?

In general we don't throw explicitly 521 errors and it is most likely caused by the edge function not having access to the PocketBase server, but I'm not sure why.

**Comment by rutikthakre** [comment]
I am using http

**Reply by ganigeorgiev** [reply]
I don't know how to help you debug it and I don't think your issue is related to PocketBase.

Are you connecting via IP or the ephemeral AWS domain name? Can you access the Admin UI with the same url in your browser?

**Reply by rutikthakre** [reply]
I am able to access admin ui and i am using same url and I think it is problem with deployment. I will redeploy. And Thanks for your time.

**Reply by ganigeorgiev** [reply]
If your Vercel frontend is on HTTPS and tries to send non-HTTPS requests then I suspect this could be the reason for the error (_I'm not sure if there is a buitlin https resources redirect which in turn could cause the 521 error_).

**Reply by rutikthakre** [reply]
Yes, it has been solved now. sending request to http on https web was the problem. thank you so much
